great_expectations/data_context/migrator/configuration_bundle.py

```diff
@@ -0,0 +1,199 @@
+"""TODO: Add docstring"""
+
+from typing import List, cast
+
+from marshmallow import Schema, fields, post_dump
+
+from great_expectations.core.expectation_suite import (
+    ExpectationSuite,
+    ExpectationSuiteSchema,
+)
+from great_expectations.core.expectation_validation_result import (
+    ExpectationSuiteValidationResult,
+    ExpectationSuiteValidationResultSchema,
+)
+from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.data_context.data_context.base_data_context import (
+    BaseDataContext,
+)
+from great_expectations.data_context.data_context_variables import DataContextVariables
+from great_expectations.data_context.types.base import (
+    CheckpointConfig,
+    CheckpointConfigSchema,
+    DataContextConfigSchema,
+    DatasourceConfig,
+    DatasourceConfigSchema,
+)
+from great_expectations.rule_based_profiler.config.base import (
+    RuleBasedProfilerConfig,
+    RuleBasedProfilerConfigSchema,
+    ruleBasedProfilerConfigSchema,
+)
```

Comment:
Are there any of these imports that are just for typing that when can move behind a `if TYPE_CHECKING` block?

Reply:
I don't believe so! I'll do a second pass with this next PR